gen/dcos-config.yaml

```diff
@@ -1333,6 +1333,50 @@ package:
   - path: /etc_master/telegraf/telegraf.d/master.conf
     content: |
       # Additional Telegraf config for masters
+      # Statsd UDP/TCP Server
+      [[inputs.statsd]]
+        ## Protocol, must be "tcp", "udp", "udp4" or "udp6" (default=udp)
+        protocol = "udp"
+        ## MaxTCPConnection - applicable when protocol is set to tcp (default=250)
+        # max_tcp_connections = 250
+        ## Enable TCP keep alive probes (default=false)
+        # tcp_keep_alive = false
+        ## Specifies the keep-alive period for an active network connection.
+        ## Only applies to TCP sockets and will be ignored if tcp_keep_alive is false.
+        ## Defaults to the OS configuration.
+        # tcp_keep_alive_period = "2h"
+        ## Address and port to host UDP listener on
+        service_address = "$DCOS_NODE_PRIVATE_IP:61825"
```

Comment:
Is this statsd server intended to receive metrics only from the node it's running on? If that's the case, should it listen on localhost instead, to avoid exposing it to the network unnecessarily?